.github/workflows/run-snapshot-generation.yml

```diff
@@ -0,0 +1,103 @@
+---
+name: Snapshot recording
+
+on:
+  pull_request:
+    types: [ opened, edited , synchronize ]
+
+permissions:
+  checks: write
+  contents: write
+  statuses: write
+  pull-requests: write
+  actions: write
+
+jobs:
+  generate_snapshots:
+    if: ${{contains(github.event.pull_request.title, '[New snapshots]')}}
+    name: Generate snapshots
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          lfs: true
+
+      - name: Set up Git LFS
```

Comment:
question/suggestion: do we need  `Set up Git LFS`  if 
```kotlin
- uses: actions/checkout@v3
        with:
          lfs: true
```
is placed?

Reply:
Thanks for that. Of course you are right 👌